subsys/bluetooth/mesh/main.c

```diff
@@ -90,7 +88,6 @@ int bt_mesh_provision(const uint8_t net_key[16], uint16_t net_idx,
 					      comp->elem_count, net_idx);
```

Comment:
cdb node must be deallocated if keys importing failed (or network creation failed, or if anything else in `bt_mesh_provision` failed).

Reply:
It is, please take a look at https://github.com/zephyrproject-rtos/zephyr/compare/main...alxelax:zephyr:fix_valid_flag#diff-721db5c8b120e58a2f2137153346b2777330b8aeb1a6fcd5be1dc9d5ada51580L170.